editor/store/reducer.js

```diff
@@ -228,6 +228,15 @@ export const editor = flow( [
 		ignoreTypes: [ 'RECEIVE_BLOCKS', 'RESET_POST' ],
 	} ),
 ] )( {
+	autosave( state = false, action ) {
```

Comment:
Why do we represent the default state as a boolean, rather than a more semantically meaningful empty value like `null` ?

Reply:
null is probably a better choice, chenged in https://github.com/WordPress/gutenberg/pull/6257/commits/193d32e36070d6a80e136d999500ee617b9d823e